packages/babel-plugin-transform-wpcalypso-async/README.md

```diff
@@ -24,9 +24,9 @@ See [Babel options documentation](http://babeljs.io/docs/usage/options/) for mor
 
 `asyncRequire` will transform to one of:
 
-- `require.ensure` if `async` plugin option is true
-- `require` if `async` plugin option is false or unset
-
+- dynamic `import()` if `async` plugin option is `true`
+- static `require` if `async` plugin option is `false` or unset
+- nothing (will be removed and no module will be imported) if the `ignore` plugin o
```

Comment:
Seems like we were intending to finish this sentence. Perhaps something like:

```suggestion
- nothing (will be removed and no module will be imported) if the `ignore` plugin option is set
```

Reply:
Haha, seems I abandoned the editor in the middle of typing 😃 Fixed.